paymentrails/configuration.py

```diff
@@ -64,11 +65,13 @@ def set_enviroment(enviroment):
         """
         Set method to change the enviroment
         """
-        if enviroment == 'production':
-            return 'https://api.trolley.com'
-        elif enviroment == 'development':
-            return  'http://api.railz.io'
-        elif enviroment == 'integration':
-           return  'http://api.local.dev:3000'
-        return enviroment
+        if enviroment == 'development':
+            env_values = dotenv_values(".env")
+
+            if len(env_values['SERVER_URL']) == 0:
+                raise Exception("Environment selected is 'development' but SERVER_URL is empty in .env")
+            else:
+                return  env_values['SERVER_URL']
+        
+        return 'https://api.trolley.com'
```

Comment:
If I understand correctly, staging and development URLs will be configured in env_values['SERVER_URL'], is that correct?  If it is, why not read 'https://api.trolley.com' from env as well?